docs/use-keyring-vault-component.md

```diff
@@ -0,0 +1,112 @@
+# Use the keyring vault component
+
+The `keyring_vault` component extends the server capabilities and provides an interface for the database with a [HashiCorp Vault](https://www.hashicorp.com/products/vault/data-protection) server to store key and secure encryption keys. 
+
+!!! admonition "See also"
```

Comment:
Can we move this See also to the bottom?

Reply:
moved